schemas/development/openservice.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/xarf/schema-discussion/master/schemas/development/openservice.schema.json",
+  "title": "XARF BOTNET",
+  "description": "A format to report open service events.",
+  "allOf": [
+    {
+      "$ref": "xarf_shared.schema.json#/XarfBase"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "Report": {
+          "allOf": [
+            {
+              "$ref": "xarf_shared.schema.json#/IpAndUrlBasedReport"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "ReportClass": {
+                  "type": "string",
+                  "enum": ["Content"]
+                },
+                "ReportType": {
+                  "type": "string",
+                  "enum": ["OpenService"]
+                },
+                "Service": {
+                  "type": "string",
+                  "description": "name of the service (e.g. redis)"
+                },
```

Comment:
The reported vulnerability could be version specific.
```suggestion
                },
                "ServiceVersion": {
                    "type" "string",
                    "description": "service version qualifier or range of impacted versions"
                },
```

Reply:
True. And I guess then it makes sense to have a shared "Service" object as stated above